1.6/Source/ResearchProgression/ResearchTracker.cs

```diff
@@ -209,14 +209,13 @@ public override void WorldComponentTick()
                             }
                         }
                     }
-
-                    if (SemiRandomResearchMod.settings.progressAddsChoice == ProgressAddsChoice.Never && additionalAvailableProjects.Any())
-                    {
-                        additionalAvailableProjects.Clear();
-                    }
-                    tickCounter = (tickCounter + 1) % tickOffset;
+                }
+                if (SemiRandomResearchMod.settings.progressAddsChoice == ProgressAddsChoice.Never && additionalAvailableProjects.Any())
+                {
+                    additionalAvailableProjects.Clear();
                 }
             }
+            tickCounter = (tickCounter + 1) % tickOffset;

```

Comment:
This was our offender for the research that stopped. Presumably all of this code was getting skipped.

Most of it appears to be checking the next available research stuff which works in my game anyway?